meshtastic/tests/test_mesh_interface.py

```
@@ -983,7 +1008,9 @@ def generate_packet_ids() -> None:
                     packet_id = iface._generate_packet_id()
                     with packet_ids_lock:
                         packet_ids.append(packet_id)
-            except Exception as e:  # noqa: BLE001 - intentionally recording thread errors
+            except (
+                Exception
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)